teuthology/task/install/bin/stdin-killer

```diff
@@ -210,8 +210,10 @@ if __name__ == "__main__":
     }
 
     if NS.setpgrp == "self":
-        os.setpgrp()
         pgrp = os.getpgrp()
+        if pgrp != os.getpid():
```

Comment:
This looks correct as EPERM errno is returned if the process calling setpgrp is the session/process group leader. Question is, what caused this failure now.

Reply:
I'm not sure but clearly under normal circumstances `stdin-killer` is not already the process group leader. Perhaps when it was wrapped with the ulimit wrapper.